_docs/1-demo/bookinfo.md

```diff
@@ -182,7 +184,8 @@ from your browser and you should see the bookinfo application `productpage`
 displayed.  Notice that the `productpage` is displayed, with no rating
 stars since `reviews:v1` does not access the ratings service.
 
-**Note**: Replace `GATEWAY_HOST_PORT` with the output of the following command:
+**Note**: Replace `GATEWAY_URL` with the output of the following command:
```

Comment:
Unless you're on IBM Bluemix, it does contain a port. Kubernetes and docker-compose commands give a port for the environment variable.

That being said, it's no big deal to change the variable name. Would it be more confusing to those running docker-compose/kubenertes to name it "GATEWAY_URL" if it also contains a port?